caravel/assets/stylesheets/explore.css

```diff
@@ -7,3 +7,11 @@
     left: 0;
     top: 0;
 }
+
+hr {
```

Comment:
I thought it was the only <hr> tag but it's not, changing the selector to `.header hr`